build.gradle

```diff
@@ -0,0 +1,118 @@
+buildscript {
+    dependencies {
+        classpath files('gradle/gradle-witness.jar')
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'witness'
+apply plugin: 'maven-publish'
+
+compileJava {
+    sourceCompatibility = 1.7
```

Comment:
Fred itself is at 1.8.
I think it would be ok to move a plugin on as well, since it runs inside fred.

Reply:
Great find! Thanks! Will do!